gcloud-java-dns/src/main/java/com/google/gcloud/dns/ChangeRequest.java

```diff
@@ -38,280 +30,163 @@
  *
  * @see <a href="https://cloud.google.com/dns/api/v1/changes">Google Cloud DNS documentation</a>
  */
-public class ChangeRequest implements Serializable {
+public class ChangeRequest extends ChangeRequestInfo {
 
-  static final Function<Change, ChangeRequest> FROM_PB_FUNCTION =
-      new Function<Change, ChangeRequest>() {
-        @Override
-        public ChangeRequest apply(com.google.api.services.dns.model.Change pb) {
-          return ChangeRequest.fromPb(pb);
-        }
-      };
   private static final long serialVersionUID = -9027378042756366333L;
-  private final List<RecordSet> additions;
-  private final List<RecordSet> deletions;
-  private final String id;
-  private final Long startTimeMillis;
-  private final Status status;
-
-  /**
-   * This enumerates the possible states of a {@code ChangeRequest}.
-   *
-   * @see <a href="https://cloud.google.com/dns/api/v1/changes#resource">Google Cloud DNS
-   * documentation</a>
-   */
-  public enum Status {
-    PENDING,
-    DONE
-  }
+  private final DnsOptions options;
+  private final String zoneName;
```

Comment:
Though zoneName is actually consistent with the param name given in the Dns API. so, I can see why you may
want to leave it consistent.  In `Storage` we used 'blob' or 'bucket' param name when given as a string.
I prefer the style in `Storage` but OK with either way.